cranelift/codegen/src/machinst/abi.rs

```diff
@@ -97,8 +97,8 @@ pub trait ABIBody {
     /// likely closely related.
     fn gen_epilogue(&self) -> Vec<Self::I>;
 
-    /// Returns the full frame size for the given function, after prologue emission has run. This
-    /// comprises the spill space, incoming argument space, alignment padding, etc.
+    /// Returns the size of a function call frame (including return address and FP) for this
+    /// function's body.
```

Comment:
This comment is less precise than the previous one, can you tweak it to tell what this number includes?